src/Interpreters/ExternalDictionariesLoader.cpp

```diff
@@ -6,6 +7,9 @@
 #include <Databases/IDatabase.h>
 #include <Storages/IStorage.h>
 #include <Common/Config/AbstractConfigurationComparison.h>
+#include <Common/Exception.h>
+#include <Common/Logger.h>
+#include <Common/logger_useful.h>
 #include <Core/Settings.h>
```

Comment:
Some includes are probably already done in other headers and therefore redundant